common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -132,6 +132,16 @@ trait ABTestSwitches {
 
   Switch(
     ABTests,
+    "ab-contributions-epic-urgency",
+    "Test 3 new variants with messaging that highlights the urgency of supporting the Guardian",
+    owners = Seq(Owner.withGithub("jranks123")),
+    safeState = Off,
+    sellByDate = new LocalDate(2017, 3, 15),
```

Comment:
After there was some confusion over the time AB tests finish, we now recommend setting the switch expiry to be a few days after the test ends - although if you don't mind your test ending a little early when you remove the switch that is up to you.  It does give you a bit of leeway to remove the switch though which is a bonus.